website/src/pages/specs/activitypub.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: ~/layouts/MdLayout.astro
+title: ActivityPub conformance
+---
+
+Apart from a few side-effects which would be quick to implement if need arised, ActivityPods is 100% compatible with ActivityPub, and supports both client-to-server and server-to-server interactions.
+
+### Outbox
+
+### Inbox
+
+### Create/Update/Delete
+
+It's possible to create, update or delete any kind of objects. Objects are automatically stored in the corresponding containers.
+
+### Follow
+
+Every ActivityStreams actor is automatically attached `as:followers` and `as:following` collections. Whenever a Follow activity is detected, it will add
+
+The `Undo > Follow` activity is also supported.
+
+### Like
+
+Every ActivityStreams actor is automatically attached a `as:liked` collection
+
+Every ActivityStreams object is automatically attached a `as:likes` collection, as soon as a Like is detected (?). If you want non-ActivityStreams objects to have a `as:likes` collection, you should add the type `as:Object`.
+
+The `Undo > Like` activity is also supported.
+
+### Announce
+
+The `as:shares` is not implemented yet. However we have a custom `apods:announces` and `apods:announcers` collections (see below).
```

Comment:
Is there a reason why?

Reply:
The `apods:announces` collection list actors with whom we have shared the resources, and `apods:announcers` list actors who have the right to share the resource. That's easier to parse than `as:shares` which is supposed to list Announce *activities*. But I may refactor this also, and maybe talk about Authorizations rather than Announces, because that's what it is in fact (authorization to see the resource).